lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1367,6 +1367,15 @@ static bool ParseIRGenArgs(IRGenOptions &Opts, ArgList &Args,
     }
   }
 
+  if (const Arg *A = Args.getLastArg(options::OPT_lto)) {
+    if (A->containsValue("llvm"))
```

Comment:
Please ensure that the match is exact otherwise this is a fatal error.  The frontend does not permit invalid arguments.

Reply:
ditto